metrics/notebooks/clusterPlots.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
```

Comment:
Nit: remove the execution count (in general the execution metadata) before committing the notebook to avoid merge conflicts.

Reply:
Done, sorry about that, I should've cleared the metadata for all cells before committing.